qa-covid19.planx-pla.net/dashboard/Public/index.html

```diff
@@ -1,25 +1,74 @@
 <html>
     <head>
-        <title>COVID19 Data Commons Sponsors</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>COVID19 Project Partners</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+        <link href="https://fonts.googleapis.com/css2?family=Anton&display=swap" rel="stylesheet">
         <link href='./main.css' rel='stylesheet' type='text/css'>
 
         <script src="https://code.jquery.com/jquery-2.2.3.min.js"></script>
     </head>
     <body>
     	<div id='header'>
             <div class='header__img-wrapper'>
-                <a href="https://qa-covid19.planx-pla.net/" target="_blank"><img src="./sponsors-logo/gitops-logo.png" height="100px" /></a>
+                <a href="https://qa-covid19.planx-pla.net" target="_blank"><img src="./sponsors-logo/chicagoland_covid19_commons.png" height="100px" /></a>
```

Comment:
could you delete the old `gitops-logo.png`

Reply:
yes